tools/pontoon/translate.md

```diff
@@ -0,0 +1,82 @@
+# Translating in Pontoon
+
+* [Translating Strings](#translating-strings)
+  * [Submit a Suggestion](#submit-a-suggestion)
+  * [Review a Suggestion](#review-a-suggestion)
+  * [Submit a Translation](#submit-a-translation-directly)
+* [Translating in the In-context Editor](#translating-in-the-wysiwyg-editor)
+
+## Translating Strings
+
+Three string actions can be performed: submit a suggestion, review a suggestion, and submit a translation directly. The level to which a user can translate a string is dependent upon their user role (Read about user roles [here](user.md)).
```

Comment:
is dependent upon -> depends on.

You can link directly on "user role", and point to users.md#user-roles (note that it's users.md, not user.md).